assets/src/data/eventespresso/graphql-client.js

```diff
@@ -1,19 +0,0 @@
-/**
- * External imports
- */
-import ApolloClient from 'apollo-boost';
-
-const { graphqlEndpoint, eejsdata: { data } } = window;
-
-const client = new ApolloClient( {
-	uri: graphqlEndpoint || '/graphql',
-	request: ( operation ) => {
-		operation.setContext( {
-			headers: {
-				'X-WP-Nonce': data.eejs_api_nonce,
-			},
-		} );
-	},
-} );
-
```

Comment:
You can get the GQL endpoint URL from that.

Reply:
Yes, updated later with those details: https://github.com/eventespresso/event-espresso-core/pull/1833/files#diff-e33616ff3b70812455391ad7eda7f428R11